pinot-plugins/pinot-stream-ingestion/pinot-kafka-base/src/main/java/org/apache/pinot/plugin/stream/kafka/KafkaJSONMessageDecoder.java

```diff
@@ -41,12 +40,13 @@
 public class KafkaJSONMessageDecoder implements StreamMessageDecoder<byte[]> {
```

Comment:
please remove this comment I had added to javadocs of all Decoder implementations regarding not using schema